api/tools/urlEncode.ts

```diff
@@ -0,0 +1,3 @@
+(() => {
+    console.log(encodeURIComponent(process.argv[2]));
+})();
```

Comment:
Was the intention of this file being used using
```
npx ts-node tools/urlEncode "<str>"
```
or did you mean to include a script in `package.json`?

Reply:
Yep that was the intended use

Reply:
Sounds good, thanks for including this 😄